src/tabset/tabset.spec.ts

```diff
@@ -41,8 +41,12 @@ function expectTabs(nativeEl: HTMLElement, active: boolean[], disabled?: boolean
 
     if (disabled[i]) {
       expect(tabTitles[i]).toHaveCssClass('disabled');
+      expect(tabTitles[i].getAttribute('aria-disabled')).toEqual('true');
+      expect(tabTitles[i].getAttribute('tabindex')).toEqual('-1');
     } else {
       expect(tabTitles[i]).not.toHaveCssClass('disabled');
+      expect(tabTitles[i].getAttribute('aria-disabled')).toEqual('false');
+      expect(tabTitles[i].getAttribute('tabindex')).toBeFalsy();
```

Comment:
This test should be stronger - testing just for falsy values would make this test pass for say value 0, which is not what we want. Please change it to compare to the exact expected value.